tests/e2e/wp/create.spec.ts

```diff
@@ -18,30 +19,57 @@ test("State user can create a Work Plan", async ({ page }) => {
   await page.waitForResponse(`**/reports/WP/${stateAbbreviation}`);
 
   // Create WP
-  await page
-    .getByRole("button", {
-      name: "Start MFP Work Plan",
-    })
-    .click();
-
-  const modal = page.getByRole("dialog");
-  await expect(modal).toBeVisible();
-  await expect(modal).toContainText("Add new MFP Work Plan");
-
-  await page.getByLabel(`${currentYear}`).click();
-  await page.getByLabel(`First reporting period (January 1 - June 30)`).click();
-  await page.getByRole("button", { name: "Start new" }).click();
-
-  // Confirm created WP is in table
-  await expect(page.getByRole("table")).toBeVisible();
-
-  const row = page.getByRole("row", { name: "Not Started" });
-  await expect(
-    row.getByRole("gridcell", {
-      name: `${stateName} MFP Work Plan ${currentYear} - Period ${firstPeriod}`,
-    })
-  ).toBeVisible();
-
-  const editButton = row.getByRole("button", { name: "Edit", exact: true });
-  await expect(editButton).toBeVisible();
+  const startMFPWorkPlanButton = page.getByRole("button", {
+    name: "Start MFP Work Plan",
+  });
+
+  const continueMFPWorkPlanButton = page.getByRole("button", {
+    name: "Continue MFP Work Plan for next Period",
+  });
+
+  // check if work plans exist already or not
+  if (await startMFPWorkPlanButton.isVisible()) {
+    await startMFPWorkPlanButton.click();
+    const modal = page.getByRole("dialog");
+    await expect(modal).toBeVisible();
+    await expect(modal).toContainText("Add new MFP Work Plan");
+
+    await page.getByLabel(`${currentYear}`).click();
+    await page
+      .getByLabel(`First reporting period (January 1 - June 30)`)
+      .click();
+    await page.getByRole("button", { name: "Start new" }).click();
+    // Confirm created WP is in table
+    await expect(page.getByRole("table")).toBeVisible();
+
+    const row = page.getByRole("row", { name: "Not Started" });
+    await expect(
+      row.getByRole("gridcell", {
+        name: `${stateName} MFP Work Plan ${currentYear} - Period ${firstPeriod}`,
+      })
+    ).toBeVisible();
+
+    const editButton = row.getByRole("button", { name: "Edit", exact: true });
+    await expect(editButton).toBeVisible();
+  } else {
+    await continueMFPWorkPlanButton.click();
+    const modal = page.getByRole("dialog");
+    await expect(modal).toBeVisible();
+    await expect(modal).toContainText("Continue");
+    await page
+      .getByRole("button", { name: "Continue from previous period" })
+      .click();
+    // Confirm created WP is in table
+    await expect(page.getByRole("table")).toBeVisible();
+
+    const row = page.getByRole("row", { name: "Not Started" });
+    await expect(
+      row.getByRole("gridcell", {
+        name: `${stateName} MFP Work Plan ${currentYear} - Period ${secondPeriod}`,
+      })
+    ).toBeVisible();
+
+    const editButton = row.getByRole("button", { name: "Edit", exact: true });
+    await expect(editButton).toBeVisible();
```

Comment:
nit: all of this code could be moved out of both if/else statements since it's duplicated. this might help clarify what the if/else is meant for (making sure we interact with/create the expected report)